internal/cli/usercmd/service.go

```diff
@@ -49,10 +53,23 @@ func (c *EpinioClient) ServiceCatalogShow(serviceName string) error {
 
 	service := catalogShowResponse.CatalogService
 
+	helmChartName := service.HelmChart
+	helmChartVersion := ""
+	pieces := strings.SplitN(helmChartName, ":", 2)
+	if len(pieces) == 2 {
+		helmChartVersion = pieces[1]
+		helmChartName = pieces[0]
+	}
+
 	c.ui.Success().WithTable("Key", "Value").
 		WithTableRow("Name", service.Name).
 		WithTableRow("Short Description", service.ShortDescription).
 		WithTableRow("Description", service.Description).
+		WithTableRow("Helm", "").
+		WithTableRow("- Repository", service.HelmRepo.URL).
```

Comment:
We discussed this in a call with @richard-cox yesterday, whether we should show this in the web UI or not. My opinion is that the repository should not be exposed to the developer. The reason is this is information only available to the operator that created the catalog service and the epinio server binary. The location of the repository may not be something the operator want to expose. The cart information below is fine though, but we should probably not call it "Chart", the developer may not even know what a chart is. Maybe we can call the column "Version" and only show a value when a version exists? Same thing in the list command above.

Reply:
Slightly related, if we obfuscate the chart concept how do we present the version, should we show the app or chart version?

Reply:
Good point, we don't have the app version available currently but that's the one the user cares about. Maybe we should store both versions and only show the app version to the user?

Reply:
Done, commit 6eef435b05a98b2f575a66cf237529c070697ba9

Reply:
Redone, commit b80a029f85b7a03cc63226099112be5a9ac5b091